src/backend/app/projects/project_routes.py

```diff
@@ -694,7 +694,7 @@ async def generate_log(
             project_id, db
         )
 
-        with open(f"/tmp/{project_id}_generate.log", "r") as f:
+        with open("/opt/logs/create_project.log", "r") as f:
```

Comment:
Once these logs are in JSON format `serialize=True`, you can parse them as JSON.

```python
with open("/opt/logs/create_project.log", "r") as log_file:
    logs = [json.loads(line) for line in log_file]
    filtered_logs = [log for log in logs if log.get("extra", {}).get("project_id") == project_id]
    last_100_logs = filtered_logs[-50:]
```

Give this a go and if it doesn't work properly we could look into it together 👍